avaframe/runScripts/runComputeDFAPath.py

```diff
@@ -22,7 +22,7 @@
 from avaframe.out3Plot import outCom3Plots
 
 # set to true if you want to run com1DFA first to create the DFA input needed for the path generation
-runDFAModule = True
+runDFAModule = False
```

Comment:
Mh, I think this should be set to True by default. Or is there a good reason for using False?